docs/section19/19-10.md

```diff
@@ -0,0 +1,72 @@
+# Section 19 - Rule 19-10       
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** True    
+**Rule ID:** 19-10       
+**Rule Description:** Air economizers shall be included in baseline HVAC Systems 3 through 8, and 11, 12, and 13 based on climate as specified in Section G3.1.2.6 with the following exceptions:   
+1. Systems that include gas-phase air cleaning to meet the requirements of Standard 62.1, Section 6.1.2. This exception shall be used only if the system in the  proposed design does not match the building design.
+2. Where the use of outdoor air for cooling will affect supermarket open refrigerated case-work systems. This exception shall only be used if the system in the  proposed design does not use an economizer. If the exception is used, an economizer shall not be included in the baseline building design.
+3. Systems that serve computer rooms complying with Section G3.1.2.6.1.
+
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE/UNDETERMINED       
+**Appendix G Section:** G3.1.2.6 including exceptions 1-3.      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:**  
+1. Applies to buildings NOT in climate zones 0A, 0B, 1A, 1B, 2A, 3A, and 4A.   
+2. Applies to baseline HVAC system types 3 through 8, and 11, 12, and 13.   
+ 
+
+**Function Calls:**  
+1. get_baseline_system_types  
+2. get_HVAC_systems_primarily_serving_comp_rooms  
+3. get_hvac_zone_list_w_area  
+4. Get_list_hvac_systems_associated_with_zone  
+5. baseline_system_type_compare
+
+## Rule Logic:   
+**Applicability Check 1 -  Check if Climate Zone Requires Economizer**  
+- Get climate zone: `climate_zone = ASHRAE229.weather.climate_zone` 
+- If climate zone is not 0A, 0B, 1A, 1B, 2A, 3A, and 4A then carry on, otherwise outcome for all HVAC systems NOT_APPLICABLE: `if climate_zone Not in ["CZ0A", "CZ0B", "CZ1A", "CZ1B", "CZ2A", "CZ3A", "CZ4A"]:`  
+    **Applicability Check 2 - Check if there are any Applicable Baseline System Types in the Project at All**   
+    - Create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_3, HVAC_SYS.SYS_4, HVAC_SYS.SYS_5,HVAC_SYS.SYS_6,HVAC_SYS.SYS_7,HVAC_SYS.SYS_8,HVAC_SYS.SYS_11,HVAC_SYS.SYS_12,HVAC_SYS.SYS_13]`
```

Comment:
Do we also want permutations of these baseline systems, e.g., 3a, 3b, etc.? YOu seem to have used these down below, so looks OK.

Reply:
@mkarpman yeah, this is an input to the function Karen created for checking this.